sre-recipes/README.md

```diff
@@ -0,0 +1,27 @@
+# SRE Recipes
+
+SRE Recipes is a tool to help users familiarize themselves with finding the root cause of a breakage using monitoring.
+
+## Usage
+
+To view which recipes exist, run the command below in this directory:
+
+```
+$ python3 sandboxctl --help
```

Comment:
You should be able to run sandboxctl directly instead of with python3 since you added the shebang, right?

Reply:
Maybe I'm doing something wrong, but I get this error when running `sandboxctl --help` (for example):
```
bash: sandboxctl: command not found
```

Reply:
Hmm does it work when you run it with `./sandboxctl --help`? I guess the problem may be that it's not on the path, so you have to call it more directly. We could pretty easily change the cloud shell image to make that directory part of the path, but that may not be worth it. Calling it with python is fine too!

Reply:
Ah, it works with `./sandboxctl --help`! I will update the README